github/ci/prow-performance-workloads/hack/common.sh

```diff
@@ -12,7 +12,14 @@ setup(){
     mkdir -p ~/.ssh && chmod 0700 ~/.ssh
     extract_secret 'prowPerformance.sshKey' ~/.ssh/id_rsa
     chmod 0600 ~/.ssh/id_rsa
+    printf "\n" >> ~/.ssh/id_rsa
```

Comment:
It was probably for debugging, but I don't think it's a good idea to print this out and make private key public....

Reply:
@marceloamaral This command won't show the private key file contents, just makes sure that it always has a new line at the end

Reply:
ohh that's right, I didn't see it very well!